browser/brave_wallet/BUILD.gn

```diff
@@ -4,6 +4,8 @@ import("//extensions/buildflags/buildflags.gni")
 assert(brave_wallet_enabled)
 
 source_set("brave_wallet") {
+  check_includes = false
```

Comment:
This is not ok and needs to be fixed asap. `check_includes = false` should never be added to new targets. @mariospr has been working hard to get rid of these and we should never add new ones